dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/company.jsp

```diff
@@ -276,3 +320,28 @@
         <%= LanguageUtil.get(pageContext, "save") %>
     </button>
 </div>
+
+<div id="regenerateKeyDialog" dojoType="dijit.Dialog" style="display:none;width:500px;vertical-align: middle; " draggable="true" title="<%= LanguageUtil.get(pageContext, "key-digest-regenerate-prompt") %>" >
+
+	<span class="ui-confirmdialog-message" >
+		<%= LanguageUtil.get(pageContext, "key-digest-regenerate-warning") %>
+	</span>
+
+    <div>
+        <table class="sTypeTable" style="width:90%; border-collapse: separate; border-spacing: 10px 15px;margin-bottom:10px;">
+          <tr>
+			<td style="width:50%;text-align: right">
+				<button id="cancelButton" dojoType="dijit.form.Button" class="dijitButton" data-dojo-props="onClick: regenerateKeyProxyCancel">
```

Comment:
Codacy found an issue: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)